spec/features/course/achievement_management_spec.rb

```diff
@@ -108,6 +108,158 @@
+
+          it 'editing an achievement condition' do
+            expect(current_path).to eq edit_course_achievement_path(course, achievement)
+            expect(page).to have_selector('tr.condition > td:nth-child(2)',
+                                          text: achievement_which_is_condition.title)
+            condition_edit_path =
+              edit_course_achievement_condition_achievement_path(course,
+                                                                 achievement,
+                                                                 achievement_condition)
+            find_link(nil, href: condition_edit_path).click
+
+            find('#condition_achievement_achievement_id').
+              find(:css, "option[value='#{some_other_achievement.id}']").
+              select_option
+            click_button 'helpers.submit.condition_achievement.update'
+            expect(current_path).to eq edit_course_achievement_path(course, achievement)
+            expect(page).to have_selector('tr.condition > td:nth-child(2)',
+                                          text: some_other_achievement.title)
+          end
+
+          it 'deleting an achievement condition' do
+            condition_delete_path =
+              course_achievement_condition_achievement_path(course, achievement,
+                                                            achievement_condition)
+            expect do
+              find_link(nil, href: condition_delete_path).click
+            end.to change { achievement.conditions.count }.by(-1)
+            expect(page).not_to have_selector('tr.condition > td:nth-child(2)',
+                                              text: achievement_which_is_condition.title)
+          end
+        end
+      end
+
+      context 'achievement level conditions' do
+        it { is_expected.to have_selector(:link_or_button,
+                                          'course.condition.levels.new.header') }
+        context 'creating a level condition' do
+          before do
+            click_link I18n.t('course.condition.levels.new.header')
+            expect(current_path).to eq(new_course_achievement_condition_level_path(course,
+                                                                                   achievement))
+            fill_in 'minimum_level', with: '10'
+            click_button 'helpers.submit.condition_level.create'
+          end
+          it 'creates a level condition correctly' do
+            expect(current_path).to eq edit_course_achievement_path(course, achievement)
+            expect(page).to have_selector('tr.condition > td:nth-child(2)', text: '10')
+          end
+        end
+
+        context 'editing and deleting' do
+          let(:level_condition) { create(:course_condition_level, course: course) }
+          let!(:achievement) { create(:course_achievement, course: course,
+                                      conditions: [level_condition]) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 190, 70 should be on its own line.